curations/nuget/nuget/-/System.Data.Odbc.yaml

```diff
@@ -0,0 +1,8 @@
+coordinates:
+  name: System.Data.Odbc
+  provider: nuget
+  type: nuget
+revisions:
+  4.5.0:
+    licensed:
+      declared: MIT
```

Comment:
https://www.nuget.org/packages/System.Data.Odbc links to https://github.com/dotnet/corefx/blob/master/LICENSE.TXT which is MIT